CHANGELOG.md

```diff
@@ -51,6 +51,13 @@ All notable changes to this project will be documented in this file.
     * #### Added
     * #### Changed
     * #### Removed
+* ### NI DIGITAL
```

Comment:
Should be NI-Digital or NI-Digital Pattern Driver (second one more correct, but definitely missing the - )